wizelineacademy/laboratoria-hacker-books
# Feat/laboratoria fe 2/searchbar component

## What does this PR do?

Feature about Searchbar component
Implemented: 
  - Consider placeholder text
  - Consider search icon
  - Attach the component to the Navbar

* Investigated and Implemented fn filter input

## Where should the reviewer start?

- Pull branch feat/laboratoria-FE-2/searchbar-component
- Run npm run dev
- Consider on Navbar the component searchbar

:checkered_flag:

## How should this be manually tested?

- Check searchbar when you write on input must show value state React
- Watch list test filter on the main section - Write search and check that filter existent names in the list

## What are the relevant tickets?

 Kanban issue [FE 2] Create Searchbar component
Closes #x

## Questions
 
Please, could you help me to review this PR  @ivanovishado and  @jonathanperazawl?

:question:
